Add (very) short introduction to Miking DPPL and CorePPL to README

dlunde opened the pull request.

david-broman commented on README.md:
````diff
@@ -1,13 +1,43 @@
+observe true (Bernoulli x);
+x
+```
+encodes a simple distribution for the bias of a coin, by setting a Beta prior for the bias and observing a single flip of the coin (resulting in `true`, i.e. heads).
````
From earlier discussions, we should probably not use the term `bias` in this context. Suggestion: "... by setting a Beta prior for the probability observing head (resulting in `true`) for a single flip of the coin."

dlunde replied:
Done, resolving

david-broman commented on README.md:
```diff
@@ -17,14 +47,14 @@ CUDA installation guides: [Linux](https://docs.nvidia.com/cuda/cuda-installation
 
-### Install rootppl
+#### Install rootppl
 To install rootppl for the current user, first clone this repository and change directory to the rootppl folder. Then run:
```
Change to `rootppl` in typewriter font.

dlunde replied:
Sure, I can change it. Note, however, that I have not made any changes to the RootPPL section (written by Joey).

david-broman commented:
I also found some things that need to be updated in the RootPPL part:

- When we talk about sweeps, it would be good to add a sentence to say what we mean.
- The RootPPL text talks about `pc`. This should be updated to describe how `next` is used.

david-broman commented:
I just discovered for `midppl`, it is not possible to write
```
make clean
make install
```

you have to write 
```
make clean
make
make install
```
it would be good if we can just write `make install` and it would then build it also.

david-broman commented:
Besides the above, it worked well to compile and run the example in the latest version.

dlunde commented:
I have now addressed the comments.

You can now run `make install` and `midppl` will be built, if not built already. The `Makefile` now has a dependency on `find` for establishing dependencies on all `.mc`-files in the repo for when to rebuild `build/midppl`. An alternative is to make `build/midppl` `PHONY`, but then it will always be rebuilt when running `make`, `make install`, etc.

Note that I have not made any modifications to the RootPPL part of the README (written by Joey). Iterating the RootPPL section would also be good, but is not part of this PR. @david-broman Maybe you can make an issue with your comments, so we don't forget?

david-broman commented:
OK, I can fix it in a later PR. Thanks!
